scripts/copyrights.json

```diff
@@ -0,0 +1,6594 @@
+[
+    {
+        "file path": "fossology-01-07-2021/utils/freshmeat/tests/runGp.php",
```

Comment:
Do we really need to store the "file path" in the json? As far as I know, it is not rendered anywhere in the UI.

Reply:
No, we don't need filepath thing, But the csv is having filepath in it, and even if I somehow remove the filepath from the json, then it won't be genralized for all the csv's, we can access the copyrights such as `cpyrightsData.copyright`

Reply:
@GMishx do we need any changes in this pr?